src/links/links.service.ts

```diff
@@ -28,20 +30,33 @@ export class LinksService {
     }
   }
 
```

Comment:
**praise**: Using `UpdateLinkDto` for updates improves maintainability and ensures data consistency.